modules/@apostrophecms/login/index.js

```diff
@@ -154,6 +154,47 @@ module.exports = {
           }
           return requirement.props(req, user);
         },
+        async requirementVerify(req) {
+          const name = self.apos.launder.string(req.body.name);
+
+          const { user } = await self.findIncompleteTokenAndUser(req, req.body.incompleteToken);
+
+          const requirement = self.requirements[name];
+
+          if (!requirement) {
+            throw self.apos.error('notfound');
+          }
+
+          if (!requirement.verify) {
+            return {};
+          }
+
+          try {
+            await requirement.verify(req, user, req.body.requirementValue);
+
+            const token = await self.bearerTokens.findOne({
```

Comment:
Not sure I need to get the token before I update it?

Reply:
I guess it's a nice check that the token makes sense. Could be helpful to someone developing, so let's keep it.